continuous_integration/recipe/meta.yaml

```diff
@@ -37,6 +39,8 @@ requirements:
     - pygments
     - nest-asyncio
     - tabulate
+    - setuptools-rust>=1.1.2
+    - pyarrow>=0.15.1
```

Comment:
If we need to add pyarrow to the conda package, I'll note that @rjzamora commented that our min version is quite low here (0.15.1 is over 2 years old now) - I'm tempted to bump to 1.0.0 since that's what's installed in for our independent cluster testing but I'll defer to him since he probably knows more than me here

Reply:
I would like to bump the version as well. After you ask which version let me know and I will change it. Our use is very simple for pyarrow, we just use it to get the Arrow Types since that is what Datafusion uses so it makes the types mapping more straight forward. I saw that to point out we likely don't have to worry about breaking changes from newer versions as those types almost never change.

Reply:
After updating it's also worth confirming it doesn't conflict with the pyarrow requirements that cuDF has and doesn't overwrite the pyarrow cuda builds that cuDF requests

Reply:
Good point

Reply:
bumped to `7.0.0` which seems to be what was present in the [cudf repo](https://github.com/rapidsai/cudf/blob/branch-22.06/conda/environments/cudf_dev_cuda11.5.yml#L21)